controllers/dynakube/updates/version.go

```diff
@@ -132,7 +151,9 @@ func updateOneAgentInstallerVersion(rec *utils.Reconciliation, dk *dynatracev1al
 		}
 	}
 
-	rec.Log.Info("OneAgent update found", "oldVersion", oldVer, "newVersion", ver)
+	logMessage := fmt.Sprintf("OneAgent update found old-version=%s new-version=%s", oldVer, ver)
+	rec.Log.Info(logMessage)
+	recorder.Event(dk, corev1.EventTypeNormal, UpdateOneAgentInstallerVersionEvent, logMessage)
```

Comment:
generally speaking, firing of events should be wrapped in functions, in this case, for example,  `fireOneAgentUpdateEvent(...)`, this way the code is easier to read